src/Idris/IBC.hs

```diff
@@ -168,9 +168,8 @@ entries i = catMaybes [Just $ toEntry "ver" 0 (encode $ ver i),
                        makeEntry "ibc_patdefs"  (ibc_patdefs i),
                        makeEntry "ibc_postulates"  (ibc_postulates i),
                        makeEntry "ibc_externs"  (ibc_externs i),
-                       case ibc_parsedSpan i of
-                            Nothing -> Nothing
-                            Just s -> Just $ toEntry "ibc_parsedSpan" 0 (encode s),
+                       ibc_parsedSpan i
+                            >>= \s -> Just $ toEntry "ibc_parsedSpan" 0 (encode s),
```

Comment:
Wouldn't this be simpler as `fmap (toEntry "ibc_parsedSpan" 0 . encode) (ibc_parsedSpan i)` or  `toEntry "ibc_parsedSpan" 0 . encode <$> ibc_parsedSpan i`?